### PR TITLE
<Feature: Move Zoom Combined>

### PR DESCRIPTION
**Pull request by Vychron**
Add Link between zooming and vertical movement

### CODE REVIEW

File: Assets/Scripts/Features/Gameplay/Camera Zoom/CameraZooming.cs [hunk 1]
```diff
@@ -16,6 +21,14 @@ public class CameraZooming : MonoBehaviour
...
     [SerializeField] private float _maxZoomDistance = 5.0f;
 
+    public float GetZoomLevel
+    {
+        get
+        {
+            return Mathf.Abs(transform.position.z - _minZoomDistance);
+        }
+    }
+
```
Comment (projojoboy) [inline Assets/Scripts/Features/Gameplay/Camera Zoom/CameraZooming.cs:34]: You need to place XML comments due to it is a public function! ლ(ಠ_ಠლ)

File: Assets/Scripts/Features/Gameplay/Camera/CameraDragging.cs [hunk 2]
```diff
@@ -91,7 +102,7 @@ private void MoveObject()
     }
     private void Update()
```
Comment (projojoboy) [inline Assets/Scripts/Features/Gameplay/Camera/CameraDragging.cs]: WHITE REGEL ** ANGRY EXHALE **  \(!!˚☐˚)/

**Review by projojoboy — CHANGES_REQUESTED** [review]
Why do you hate me

File: Assets/Scripts/Features/Gameplay/Camera/CameraDragging.cs (hunk 2 above)
Comment (Vychron) [reply Assets/Scripts/Features/Gameplay/Camera/CameraDragging.cs]: Your comment was not completely in English.

Comment (projojoboy) [reply Assets/Scripts/Features/Gameplay/Camera/CameraDragging.cs]: I do not know wet jij aan het praten bent

**Review by projojoboy — APPROVED** [review]

**Pull request merged**